tests/unit/exporters/test_wandb_data_exporter.py

```
@@ -0,0 +1,285 @@
+def fake_wandb(monkeypatch: pytest.MonkeyPatch) -> dict[str, Any]:
+    """Install a fake wandb module into sys.modules and return call state."""
+    state: dict[str, Any] = {
+        "init_kwargs": None,
+        "logged": [],
+        "artifact_files": [],
+        "logged_artifacts": [],
+        "finished": False,
+    }
+
+    class FakeArtifact:
+        def __init__(self, name: str, type: str) -> None:
```

Ruff (0.15.15)

[error] Function argument `type` is shadowing a Python builtin

(A002)